wiki/Glossary/en.md

```diff
@@ -156,7 +162,7 @@ Collab (or collaboration) has two meanings:
    - this is to ensure that the creator has done enough to upload the map under their username
 2. the mapset containing guest difficulties
 
-Collab may sometimes be used in difficulty version names.
+Collab may some times be used in difficulty version names.
```

Comment:
spacing fix from Replace All misfire.

`Collab may sometimes be used in difficulty version names.`